q2_annotate/busco/types/_format.py

```diff
@@ -132,6 +132,21 @@ class BuscoDatabaseDirFmt(model.DirectoryFormat):
         r'busco_downloads\/lineages\/.+refseq_db\.faa\.gz',
         format=BuscoGenericBinaryFileFmt
     )
+    information = model.FileCollection(
+        r'busco_downloads\/information\/.+\.txt$',
+        format=BuscoGenericTextFileFmt,
+        optional=True
+    )
+    missing_parasitic = model.File(
+        r'busco_downloads\/lineages\/fungi_odb10\/missing_in_parasitic\.txt$',
+        format=BuscoGenericTextFileFmt,
+        optional=True
+    )
+    no_hits = model.File(
+        r'busco_downloads\/lineages\/pectobacteriaceae_odb12\/no_hits$',
+        format=BuscoGenericTextFileFmt,
+        optional=True
+    )
```

Comment:
I think this should also be generalized, in case it happens in the future, by replacing the _pectobacteriaceae_ lineage with a wildcard.